src/modules/projects/components/tables/ProjectExternalSubmissionsTable.tsx

```diff
@@ -132,6 +144,7 @@ const ProjectExternalSubmissionsTable = ({
     getVariables: (values) => {
       return {
         id: selected?.id || '', // selected should never be undefined when dialog is open
```

Comment:
```suggestion
        id: selectedId || '', // selected should never be undefined when dialog is open
```
`selected` is null while useGetExternalFormSubmissionQuery is fetching. So it's possible to hit submit while the view is loading, which results in an error